remill/Arch/AArch64/Arch.cpp

```diff
@@ -4656,6 +4656,39 @@ bool TryDecodeMVNI_ASIMDIMM_M_SM(const InstData &data, Instruction &inst) {
   return true;
 }
 
+// USHR  <V><d>, <V><n>, #<shift>
+bool TryDecodeUSHR_ASISDSHF_R(const InstData &data, Instruction &inst) {
+  if ((data.immh.uimm & 8) == 0) {
```

Comment:
Shouldn't this be `if (0 != (data.immh.uimm & 8) && !data.Q)`?

Reply:
The logic here is correct, but I copied the wrong comment from the documentation. Fixed in c6035e3.